dev/checkstyle.xml

```diff
@@ -196,6 +196,17 @@
             <property name="format" value="@Test\(expected"/>
             <property name="message" value="Please use the `assertThrows` method to test for exceptions." />
         </module>
+        <module name="RegexpSinglelineJava">
```

Comment:
Add new rules to align the `scala side`: `scalastyle-config.xml`

Reply:
The rule on the `java side` is `slightly different` from `scala side` because the following code is not allowed on the java side:
```
xxx.toUpperCase // no ()
xxx.toLowerCase // no ()
```